### PR TITLE
docs(support): feedback from dev team on issues + support.md correction

### PR DESCRIPTION
**Pull request by Valastiri**
- Corrected altitude issues as referred by donbikes
- Updated screenshot section for grammar and removed `:` from h4 elements

### CODE REVIEW

File: docs/start/reported-issues.md [hunk 1]
```diff
@@ -112,16 +110,17 @@ FBW Installer - [Download Here](https://api.flybywiresim.com/installer) / *Sim V
 
 ### Altitude Issues
 
-The effects of non-standard day pressure and temperature on altitude in MSFS is inaccurate for SU5 affecting the following:
-
-- Airplane altitude provided to ATC from the airplane transponder will not correlate correctly with the airplane indicated on the airplane's altimeter. VATSIM / IVAO will see MSFS airplanes at a different altitude than what the pilots see.
-
-- If temperature is different than ISA, the airplane altimeter will not indicate the correct altitude, most observable when airplane is on the ground at the airport with the proper QNH set. The altimeter should align with the airport's elevation, but it won't with SU5.
+The effects of non-standard day pressure and temperature on altitude in MSFS are inaccurate for SU5 affecting the following:
```
Comment (frankkopp) [inline docs/start/reported-issues.md:113]: ...non-standard day pressure and temperature on altitude...

Is the "day" correct? What is a day pressure?

File: docs/start/support.md [hunk 2]
```diff
@@ -145,13 +145,13 @@ Make a screenshot of the Community Folder if requested by our support team.
 
 Quite often when issues during flight are reported a screenshot of the Cockpit will be request by our support team. A good screenshot should contain all screens and the FCU in a sufficient resolution to help the support team to recognize details on the PFD, ND or ECAMs.
 
-#### Example:
+#### Example
 
 ![Cockpit screenshot for Support](../assets/support-guide/cockpit-screenshot.jpg "Cockpit screenshot for Support"){loading=lazy}
 
-#### How to do a good screenshot:
+#### How to Take a Good Screenshot
```
Comment (frankkopp) [inline docs/start/support.md:152]: Capital "Take" or "take"?

**Review by frankkopp — CHANGES_REQUESTED** [review]

File: docs/start/support.md (hunk 2 above)
Comment (Valastiri) [reply docs/start/support.md:152]: discussed in support ops

File: docs/start/reported-issues.md (hunk 1 above)
Comment (Valastiri) [reply docs/start/reported-issues.md:113]: Looked it up on discord as well. Should be fine!

**Pull request merged**